src/core/sent_packet_metadata.h

```diff
@@ -74,6 +74,10 @@ typedef struct QUIC_SENT_FRAME_METADATA {
 
 } QUIC_SENT_FRAME_METADATA;
 
+CXPLAT_STATIC_ASSERT(
+    QUIC_FRAME_MAX_SUPPORTED <= (uint64_t)UINT8_MAX,
+    "Metadata 'Type' field above assumes frames types fit in 8-bits");
```

Comment:
This only applies for packets that are sent, right? Not received?

Reply:
Correct. It's "sent" metadata.